README.md

```diff
@@ -36,6 +36,10 @@ An implementation of a parser for the Password Rules language that's written in
 
 The file [`quirks/websites-with-shared-credential-backends.json`](quirks/websites-with-shared-credential-backends.json) contains a list of groups of websites that are known to share the same credential backend. This can be used to offer contextually relevant accounts to users on website A, even if credentials were previously saved for website B.
```

Comment:
I think we can achieve your goal here by amending this paragraph. Here’s my attempt, incorporating some of your great phrasing here.

> The file [`quirks/websites-with-shared-credential-backends.json`](quirks/websites-with-shared-credential-backends.json) contains a list of groups of websites that share the same credential backend and serve pages where users can sign in, accepting accounts from the others. For example, adding `first.website` and `second.website` means that `first.website` and `second.website` each serve a page (e.g. `first.website/login` and `second.website/login`) where the same accounts are valid for signing in, despite the different domains. It wouldn’t be appropriate to associate `google.com.li` to `google.com` because `google.com.li` redirects to `accounts.google.com` for sign-in, and `google.com.li` never serves a login page.
> 
> This data can be used by password managers to offer contextually relevant accounts to users on first.website, even if credentials were previously saved for second.website. This list should not be used as part of any user experience that releases...

Reply:
thanks! 🤝